githttp.go

```diff
@@ -10,6 +10,14 @@ import (
 	"strings"
 )
 
+type ResolveInfo struct {
+	Path    string
+	Request *http.Request
```

Comment:
The repo path is extracted from a regex when it looks up the service, since the full request path might have /info/refs or other things after it, so the path of the repo and the path of the request will be different.